install

```diff
@@ -37,7 +37,7 @@ pip install https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.
 sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
 sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net --recv-key 0xB01FA116
 sudo apt-get update -qq
-sudo apt-get install -y python-catkin-pkg python-rosdep ros-indigo-catkin ros-indigo-ros ros-indigo-roslaunch build-essential
+sudo apt-get install -y python-catkin-pkg python-rosdep ros-indigo-catkin ros-indigo-ros ros-indigo-roslaunch build-essential python3-tk
```

Comment:
as a note. I don't think having this here is bad, but I also don't know if the problems we were running into ( #129 #126 ) are fixed by doing this. I think the problems are being caused by something else so having this here might be unnecessary in the long run. I think it's fine to leave it here for now though